vizro-core/src/vizro/models/_components/container.py

```diff
@@ -103,3 +117,38 @@ def _build_inner_layout(self):
             components_container[f"{self.layout.id}_{component_idx}"].children = component.build()
 
         return components_container
+
+    def _build_container(self):
+        """Returns collapsible container."""
+        return (
+            dbc.Collapse(
+                id=f"{self.id}_main_container",
+                children=self._build_inner_layout(),
+                is_open=self.collapse,
+                className="collapsible-container",
+            )
+            if self.collapse is not None
+            else self._build_inner_layout()
```

Comment:
Very minor, but can we do in the order `if self.collapse is None` instead please?